Deliverable 1/API Examples/reportSearchResults.json

```diff
@@ -0,0 +1,15 @@
+{
+  "UI_UX_Request": {
+    "query": "RPI Professors",
+    "results": [
```

Comment:
Do we want to do this separately from the `reportInteractions` call? I feel like it would just make it less useful if we don't have all of the information at once

Reply:
We can merge them and have `reportInteractions` list all search results